src/pages/patch/patchTabs/DownstreamTasks.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Table } from "antd";
+import { Patch } from "gql/generated/types";
+import { DownstreamProjectAccordion } from "./DownstreamProjectAccordion";
+
+type childPatchesType = Patch["childPatches"];
+type childPatch = childPatchesType[0];
+
+interface DownstreamTasksProps {
+  childPatches: childPatchesType;
+}
+
+export const DownstreamTasks: React.FC<DownstreamTasksProps> = ({
+  childPatches,
+}) => {
+  const columns = [
+    {
+      title: <span data-cy="child-patch">Child Patch</span>,
+      render: (
+        text: string,
+        { project, patchID, status, taskCount }: childPatch
+      ): JSX.Element => (
+        <DownstreamProjectAccordion
+          key={`downstream_project_${patchID}`}
+          projectName={project}
+          status={status}
+          childPatchId={patchID}
+          taskCount={taskCount}
+        />
+      ),
+    },
+  ];
+
+  return (
+    <div>
+      <Table
```

Comment:
Do we need a table? it looks like we are removing all the table features from it. I think it would make more sense just to iterate over the childPatches and display the accordians.

Reply:
It helps create rows. We don't specifically need it but it makes it easier.